contributing/code/bc.rst

```diff
@@ -310,6 +314,10 @@ Change return type                                  Yes             Yes
 **Static Methods**
 Turn non static into static                         No              No
 Turn static into non static                         No              No
+**Constants**
+Add constant                                        Yes             Yes
+Remove constant                                     No              No
+Change value of a constant                          Yes             Yes
```

Comment:
changing the value of a constant depends on what they are used for: constants used for event names cannot have their value changed for instance, because someone registering a listener in Yaml or XML would have to use the value directly, not the constant